mithril-common/src/crypto_helper/types/protocol_key.rs

```diff
@@ -285,4 +287,25 @@ mod test {
 
         assert_eq!(expected_key, read_key);
     }
+
+    #[test]
+    fn can_read_a_verification_key_from_file_with_trailing_whitespaces() {
+        let expected_key: ProtocolKey<StmVerificationKeyPoP> = VERIFICATION_KEY.try_into().unwrap();
+        let key_path = std::env::temp_dir().join("can_read_and_write_to_file_a_verification_key");
```

Comment:
Wouldn't it be more robust to use the dedicated temporary directory creation tooling?
```suggestion
        let key_path = TempDir::create("protocol_key", "can_read_a_verification_key_from_file_with_trailing_whitespaces");
```

But I see that it is not used in the previous tests, maybe it's not relevant.

Reply:
Done, also for remaining tests not using the correct tooling